src/drawscene.cpp

```diff
@@ -471,6 +474,42 @@ void draw_plain(Camera &camera, bool show_hud, Hud &hud,
 	}
 }
 
+inline int scaledown(int coef, int size)
+{
+	return (size + coef - 1) / coef;
+}
+
+void draw_undersampled(Camera& camera, bool show_hud,
+		Hud& hud, video::IVideoDriver* driver,
+		scene::ISceneManager* smgr, const v2u32& screensize,
+		bool draw_wield_tool, Client& client, gui::IGUIEnvironment* guienv,
```

Comment:
No, you should do it for the portions that you have changed only. Don't clean up code you are not modifying.